app/models/katello/concerns/organization_extensions.rb

```diff
@@ -86,6 +86,16 @@ def manifest_history
           imports.map { |i| OpenStruct.new(i) }
         end
 
+        def manifest_expired?
+          manifest_expiry = owner_details.dig(:upstreamConsumer, :idCert, :serial, :expiration)
```

Comment:
I did some digging and found out that my `manifest_expiry` is in July, so it's not actually expired yet.  Perhaps my manifest is broken in some other way?  I'm still seeing the errors I mentioned above.  `pry` has shown that the PR is loaded in.